application/src/main/java/org/togetherjava/tjbot/commands/BotCommandAdapter.java

```diff
@@ -88,21 +88,20 @@ public final void acceptComponentIdGenerator(ComponentIdGenerator generator) {
 
     @SuppressWarnings("NoopMethodInAbstractClass")
     @Override
-    public void onButtonClick(ButtonInteractionEvent event, List<String> args) {
+    public void onButtonInteraction(ButtonInteractionEvent event, List<String> args) {
```

Comment:
i think it was better before. `onButtonClicked`. `onSelectionMenuSubmitted` and `onModalSubmitted` are maybe good candidates.

Reply:
I don't like onSelectMenuSubmitted because you don't, press "submit" or something.

But sure, I'll name it SelectMenu though, as they're select menu's now.

Reply:
`onSelectMenuSelection` maybe? trying to figure out a good verb that describes what the user does. but yeah. i just want a name that feels natural and easy to understand to an user. `onButtonClicked` is very clear and straightforward. for modals the `onModalSubmitted` also makes a lot of sense.

Reply:
i'd say keep what was before or what it is now, changing it will make things more confusing